plugins/restlet1x/nexus-restlet1x-plugin/src/main/java/org/sonatype/nexus/rest/repositorystatuses/RepositoryStatusesListPlexusResource.java

```diff
@@ -76,6 +84,15 @@ public PathProtectionDescriptor getResourceProtection() {
   public Object get(Context context, Request request, Response response, Variant variant)
       throws ResourceException
   {
+    if (adminOnly) {
+      Subject subject = ThreadContext.getSubject();
+      // skip building reply unless user has admin access (ie. can update status)
+      if (subject == null || !subject.isPermitted("nexus:repostatus:update")) {
```

Comment:
it's part of the built-in privileges: https://github.com/sonatype/nexus-oss/blob/nexus-2.11.x/components/nexus-core/src/main/resources/META-INF/nexus/static-security.xml#L1259 - admin users will have this privilege because they inherit anything under "nexus:*".

Since status is of greatest importance to those users who can do something about it, it made more sense to use this privilege to control the behaviour. Otherwise you could end up with a user who doesn't have the "nx-admin" role, but has permission to change the status - and is blocked from seeing what the current status actually is.